src/main/java/io/milvus/response/SearchResultsWrapper.java

```diff
@@ -61,7 +61,6 @@ public List<QueryResultsWrapper.RowRecord> getRowRecords(int indexOfTarget) {
         List<IDScore> idScore = getIDScore(indexOfTarget);
         for (int i = 0; i < topK; ++i) {
             QueryResultsWrapper.RowRecord record = new QueryResultsWrapper.RowRecord();
-            record.put("id", idScore.get(i).getLongID());
```

Comment:
I thought we should not remove this info.
but we should get ID based on the schema of the collection.

SomeLike
if useStringPK {
  record.put("id", idScore.get(i).GetStringID());
} else {
  record.put("id", idScore.get(i).getLongID());

}